cameo/strain_design/core.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2015 Novo Nordisk Foundation Center for Biosustainability, DTU.
```

Comment:
There are no docstrings in this module, for classes or methods. Adding some would make it more clear what the different classes are meant to be used for.